style-guidelines/Swift.md

```diff
@@ -370,11 +381,17 @@ textContainer?.textLabel?.setNeedsDisplay()
 Use optional binding when it's more convenient to unwrap once and perform multiple operations:
 
 ```swift
-if let textContainer = self.textContainer {
+if let textContainer = textContainer {
   // do many things with textContainer
 }
 ```
 
+Use guard unwrapping if the object is required for continuing the operation.
```

Comment:
We could argue that if you are for instance in the middle of the method and you only want to use it for one thing, though.